norse/benchmark/norse_lif.py

```diff
@@ -29,7 +32,7 @@ def forward(
 
         for ts in range(sequence_length):
             x = self.fc(input_spikes[ts])
-            z, s = _lif_feed_forward_step_jit(input_tensor=x, state=s, p=p, dt=self.dt)
```

Comment:
why this change?

Reply:
I took the liberty to remove the checks for C++ availability in the functional API. Here, I'm directly calling the compiled C++ version. Would you suggest a different approach?